extension/chrome/elements/pgp_block.ts

```diff
@@ -292,48 +292,65 @@ Catch.try(async () => {
     setFrameColor(isEncrypted ? 'green' : 'gray');
     renderPgpSignatureCheckResult(sigResult);
     const publicKeys: string[] = [];
+    let renderableAtts: Att[] = [];
     let decryptedContent = decryptedBytes.toUtfStr();
+    let isHtml: boolean = false;
     // todo - replace with PgpMsg.fmtDecrypted
     if (!Mime.resemblesMsg(decryptedBytes)) {
       const fcAttBlocks: MsgBlock[] = [];
       decryptedContent = PgpMsg.extractFcAtts(decryptedContent, fcAttBlocks);
       decryptedContent = PgpMsg.stripFcTeplyToken(decryptedContent);
       decryptedContent = PgpMsg.stripPublicKeys(decryptedContent, publicKeys);
-      if (publicKeys.length) {
-        BrowserMsg.send.renderPublicKeys(parentTabId, { afterFrameId: frameId, publicKeys });
-      }
-      await renderContent(Xss.escapeTextAsRenderableHtml(decryptedContent), false);
       if (fcAttBlocks.length) {
-        renderInnerAtts(fcAttBlocks.map(attBlock => new Att(attBlock.attMeta!)));
-      }
-      if (passwordMsgLinkRes && passwordMsgLinkRes.expire) {
-        renderFutureExpiration(passwordMsgLinkRes.expire);
+        renderableAtts = fcAttBlocks.map(attBlock => new Att(attBlock.attMeta!));
       }
     } else {
       renderText('Formatting...');
       const decoded = await Mime.decode(decryptedBytes);
       if (typeof decoded.html !== 'undefined') {
-        await renderContent(decoded.html, false);
+        decryptedContent = decoded.html;
+        isHtml = true;
       } else if (typeof decoded.text !== 'undefined') {
-        await renderContent(Xss.escapeTextAsRenderableHtml(decoded.text), false);
-      } else {
-        await renderContent(Xss.escapeTextAsRenderableHtml(decryptedContent), false);
+        decryptedContent = decoded.text;
       }
-      const renderableAtts: Att[] = [];
       for (const att of decoded.atts) {
         if (att.treatAs() !== 'publicKey') {
           renderableAtts.push(att);
         } else {
           publicKeys.push(att.getData().toUtfStr());
         }
       }
-      if (renderableAtts.length) {
-        renderInnerAtts(decoded.atts);
+    }
+    if (isHtml) {
+      const $message = $('<div>').html(Xss.htmlSanitize(decryptedContent)); // xss-safe-factory
```

Comment:
I like how you distinguish jQuery variables with `$variable`. I think it adds some clarity and it's succinct. But it's inconsistent with the rest of the codebase, we don't use it anywhere else. Good things are worth doing consistently.

I suggest you rename these to not use the $ anymore, for consistency. Then create an issue to suggest to refactor variables in a similar fashion across the codebase, to do sometime later.

Reply:
Yes, I agree. I will create an issue.

It was just a habit, on previous projects that I was working on we called variables this way.